src/scripts/main.js

```diff
@@ -1,3 +1,32 @@
 'use strict';
 
-// write code here
+const list = document.querySelector('ul');
+
+function sortList(listOfElements) {
+  const itemElements = listOfElements.querySelectorAll('li');
+
+  const sortedElements = [...itemElements].sort(
+    (element1, element2) => +element2.dataset.salary.replace(/\$|,/g, '')
+      - +element1.dataset.salary.replace(/\$|,/g, '')
```

Comment:
`.dataset.salary.replace(/\$|,/g, '')` is duplicated, so make sense to move it to the separate function